src/overlays/actors/ovl_En_Bee/z_en_bee.c

```diff
@@ -67,34 +69,219 @@ static DamageTable D_80B5B214 = {
     /* Powder Keg     */ DMG_ENTRY(1, 0xF),
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80B5B234 = {
-    { COLTYPE_NONE, AT_ON | AT_TYPE_ENEMY, AC_ON | AC_HARD | AC_TYPE_PLAYER, OC1_ON, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0xF7CFFFFF, 0x08, 0x02 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_ENEMY,
+        AC_ON | AC_HARD | AC_TYPE_PLAYER,
+        OC1_ON,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0xF7CFFFFF, 0x08, 0x02 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 6, 13, -4, { 0, 0, 0 } },
 };
 
-#endif
+void EnBee_Init(Actor* thisx, PlayState* play) {
+    EnBee* this = THIS;
+
+    this->actor.colChkInfo.mass = 0xA;
```

Comment:
We use dec for mass iirc
```suggestion
    this->actor.colChkInfo.mass = 10;
```